src/main/java/Duke.java

```diff
@@ -1,10 +1,387 @@
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.lang.Exception;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.charset.StandardCharsets;
+import java.time.LocalDateTime;
+
+class DukeException extends Exception {
+    public DukeException(String s) {
+        super(s);
+    }
+}
+
 public class Duke {
+    private ArrayList<Task> todoList;
+    File saveList;
+    FileWriter editor;
+
+    public Duke() {
+        this.todoList = new ArrayList<Task>();
+    }
+
+    private void initialize(){
+        saveList=new File("src/main/data/list.txt");
```

Comment:
savelist=new violates the coding style.
Refer to https://nus-cs2103-ay1920s1.github.io/website/coding-standards/java/intermediate.html